### DISCUSSION TITLE
[VOTE] Release Apollo version 2.2.0

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#5040] [General]
Hi All,

@apolloconfig/pmc @apolloconfig/committers This is a call for vote to release Apollo version 2.2.0.

Release Note:

 * https://github.com/apolloconfig/apollo/releases/tag/v2.2.0

Release Candidate:

 * https://github.com/apolloconfig/apollo/releases/download/v2.2.0/apollo-configservice-2.2.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.2.0/apollo-adminservice-2.2.0-github.zip
 * https://github.com/apolloconfig/apollo/releases/download/v2.2.0/apollo-portal-2.2.0-github.zip
 * sha1 checksums
   - 126fec918ded7e1aae2d52d37ac3c54bba0de8b3  apollo-configservice-2.2.0-github.zip
   - 1390fdeafe26f2f5dfb2f031d513c0468cfe5e13  apollo-adminservice-2.2.0-github.zip
   - e01b2307e300eb3655d9739c4ae90ed23fa1813c  apollo-portal-2.2.0-github.zip

Release Tag :

 * https://github.com/apolloconfig/apollo/releases/tag/v2.2.0

Release CommitID :

 * https://github.com/apolloconfig/apollo/commit/ee1517be717283ae5bbb990fd2c5a809bd841b4b

Guide to build the release from source:

 * https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2212-building-from-source-code

Voting will start now (2023/12/10) and will remain open for at least 72 hours, Request all PMC members and committers to give their vote.
[ ] +1 Release this package.
[ ] +0 No opinion.
[ ] -1 Do not release this package because....

**Comment by klboke** [comment]
+1

**Comment by hezhangjian** [comment]
+1

**Comment by zouyx** [comment]
+1 approve

**Comment by nisiyong** [comment]
+1

**Comment by vdiskg** [comment]
+1

**Comment by Anilople** [comment]
+1

---

My sha1sum check

```
1390fdeafe26f2f5dfb2f031d513c0468cfe5e13  apollo-adminservice-2.2.0-github.zip
126fec918ded7e1aae2d52d37ac3c54bba0de8b3  apollo-configservice-2.2.0-github.zip
e01b2307e300eb3655d9739c4ae90ed23fa1813c  apollo-portal-2.2.0-github.zip
```

And `config/application.properties` in .zip

There are

```properties
# true: enabled the new feature of audit log
# false/missing: disable it
apollo.audit.log.enabled = true
```

In apollo-adminservice-2.2.0-github.zip and apollo-portal-2.2.0-github.zip, but not in apollo-configservice-2.2.0-github.zip,
same as expected.

**Comment by nobodyiam** [comment]
The vote to release Apollo version 2.2.0 has passed with six +1 and zero -1.
